README.md

```diff
@@ -10,7 +10,7 @@ pip install -r requirements.txt
 ```
 
 ## run `sampleAcquis.py` example: 
-```
+```shell
```

Comment:
It would seem this change belongs to the commit before ("code-blocks") rather than to this commit ("add instruction")?